frontend/src/components/molecules/LeftHeaderMenu/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { CSSProperties } from "react";
+import { Link } from "react-router-dom";
+
+const Styles: { [key: string]: CSSProperties } = {
+  leftHeaderMenu: {
+    height: "100%",
+    display: "flex",
+    alignItems: "center",
+  },
+  navLink: {
+    padding: "10px",
+    textDecoration: "none",
+    color: "inherit",
+  },
+};
+
+const navItems: { text: string; link: string }[] = [
+  { text: "ホーム", link: "/" },
+  { text: "目標一覧", link: "/" },
+  { text: "定期レポート一覧", link: "/" },
+];
+
+export const LeftHeaderMenu = () => {
+  return (
+    <nav style={Styles.leftHeaderMenu}>
+      {navItems.map((item) => (
+        <Link to={item.link} style={Styles.navLink}>
+          {item.text}
+        </Link>
+      ))}
+    </nav>
```

Comment:
これ今って警告出ないんだっけ～ブラウザコンソールとかでwarn出る気がする
mapするDOM要素には一意なidをつけよう

```suggestion
      {navItems.map((item, index) => (
        <Link id={index} to={item.link} style={Styles.navLink}>
          {item.text}
        </Link>
      ))}
    </nav>
```

Reply:
警告出てた
idじゃなくてkeyプロパティを設定しろ的なメッセージだったからそうした！
https://github.com/bc-dev-club/habittracker/pull/3/commits/fc024a67b7866e010caccd53a6b43421a5de7e0a